src/components/UserComponents/UserDisplay.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { UserSchema } from "../../schemas/UserSchema";
+import { SizeType } from "../../constants";
+import { Card, CardBody, Avatar, Text } from "@chakra-ui/react";
+import {
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverBody,
+  PopoverArrow,
+  PopoverCloseButton,
+} from "@chakra-ui/react";
+
+interface AuthorProps {
+  user: UserSchema;
+  size: SizeType;
+}
+
+export function AuthorComponent({ user, size }: AuthorProps) {
```

Comment:
might be helpful to have a small comment here just explaining what this is actually for/what it returns